domain/src/main/java/org/oppia/domain/oppialogger/exceptions/ExceptionsController.kt

```diff
@@ -10,103 +10,81 @@ import org.oppia.util.data.DataProviders
 import org.oppia.util.logging.ConsoleLogger
 import org.oppia.util.logging.ExceptionLogger
 import org.oppia.util.networking.NetworkConnectionUtil
-import org.oppia.util.system.OppiaClock
 import javax.inject.Inject
 
+const val EXCEPTIONS_CONTROLLER = "Exceptions Controller"
```

Comment:
Prefer to make this ``private`` if it isn't being accessed outside of this class.

Reply:
Done.